runbotics-orchestrator/src/main/java/com/runbotics/service/criteria/ProcessCriteria.java

```diff
@@ -43,6 +36,8 @@ public class ProcessCriteria implements Serializable, Criteria {
 
     private StringFilter tagName;
 
+    private UUIDFilter collectionId;
```

Comment:
isn't it just UUID ?

Reply:
In criteria everything is filter type as far as I know.